backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/CreateAllTemplateDisksFromSnapshotCommand.java

```diff
@@ -43,6 +43,7 @@ public CreateAllTemplateDisksFromSnapshotCommand(T parameters, CommandContext cm
         List<DiskImage> disksFromDb =
                 DisksFilter.filterImageDisks(vmFromConfiguration.getDiskMap().values(), ONLY_SNAPABLE, ONLY_ACTIVE);
         disksFromDb.addAll(DisksFilter.filterCinderDisks(getVm().getDiskMap().values(), ONLY_PLUGGED));
+        disksFromDb.addAll(DisksFilter.filterManagedBlockStorageDisks(getVm().getDiskMap().values(), ONLY_PLUGGED));
```

Comment:
I think it's good to have this code here as well but would the change in AddVmTemplateFromSnapshotCommand be sufficient for this change, i.e., to block the creation of the template? can you please point me to the validation that would fail?

Reply:
the line in CreateAllTemplateDisksFromSnapshotCommand isn't critical for blocking the operation, I just thought it should be with the same funcionality.
Do you think it's better to remove it?

AddVmTemplateFromSnapshotCommand#validate() > super.validate() > AddVmTemplateCommand#validate() > validateImages() > (!validate(storageDomainsValidator.isSupportedByManagedBlockStorageDomains(getActionType())))